### PR TITLE
Fix crash into subscriber detail

### PR DESCRIPTION
**Pull request by elabcius5**
This PR solves a crash when I click into a subscriber in Subscriber List and this doesn't have name

### CODE REVIEW

**Comment by juliocatano** [comment]
LG

File: app/src/main/java/com/github/globant/githubsubscribers/subscribersdetail/view/SubscriberDetailFragment.java [hunk 1]
```diff
@@ -156,7 +156,7 @@ public void showRepositoryError() {
 
     public void showSubscriberDetails(User userInfo) {
         Picasso.with(getContext()).load(userInfo.getAvatarUrl()).into(profileImage);
-        profileFullName.setText(Utils.setUnderlineText(userInfo.getName()));
+        profileFullName.setText(userInfo.getName() != null ? Utils.setUnderlineText(userInfo.getName()) : userInfo.getName());
```
Comment (lucaslabs) [inline app/src/main/java/com/github/globant/githubsubscribers/subscribersdetail/view/SubscriberDetailFragment.java]: This should be `userInfo.getName() != null ? Utils.setUnderlineText(userInfo.getName()) : ""`

Comment (edwin-pixel-1) [reply app/src/main/java/com/github/globant/githubsubscribers/subscribersdetail/view/SubscriberDetailFragment.java]: I suggest all this logic `userInfo.getName() != null ? Utils.setUnderlineText(userInfo.getName()) : ""` should be assigned to a new variable before to send it into setText method.

File: app/src/main/java/com/github/globant/githubsubscribers/subscribersdetail/view/SubscriberDetailFragment.java [hunk 2]
```diff
@@ -154,8 +155,9 @@ public void showUserError(int messageId) {
     }
 
     public void showSubscriberDetails(User userInfo) {
+        String name = (userInfo.getName() != null ? Utils.setUnderlineText(userInfo.getName()) : EMPTY_SPACE).toString();
```
Comment (lucaslabs) [inline app/src/main/java/com/github/globant/githubsubscribers/subscribersdetail/view/SubscriberDetailFragment.java]: Why the use of `toString`?

File: app/src/main/java/com/github/globant/githubsubscribers/commons/utils/Utils.java [hunk 3]
```diff
@@ -15,10 +15,10 @@
  */
 public class Utils {
 
-    public static SpannableString setUnderlineText(String someText) {
+    public static String setUnderlineText(String someText) {
         SpannableString spanString = new SpannableString(someText);
         spanString.setSpan(new UnderlineSpan(), 0, spanString.length(), 0);
-        return spanString;
+        return spanString.toString();
```
Comment (lucaslabs) [inline app/src/main/java/com/github/globant/githubsubscribers/commons/utils/Utils.java:21]: Nice!

**Comment by lucaslabs** [comment]
LG

**Pull request merged**